packages/dashboard-frontend/src/store/ServerConfig/index.ts

```diff
@@ -86,8 +86,7 @@ const unloadedState: State = {
     timeouts: {
       inactivityTimeout: -1,
       runTimeout: -1,
-      startTimeout: 300,
-    },
```

Comment:
I'm not sure you need to remove this timeout; it was added as part of this [PR](https://github.com/eclipse-che/che-dashboard/pull/694). However, please check it and revert this change if necessary.

Reply:
I have fixed it.